core/src/main/scala/org/ensime/core/Analyzer.scala

```diff
@@ -51,7 +54,9 @@ class Analyzer(
   private var settings: Settings = _
   private var reporter: PresentationReporter = _
 
-  protected var scalaCompiler: RichCompilerControl = _
+  protected var scalaCompiler: Option[RichCompilerControl] = _
```

Comment:
That would be nice. I seem to recall scala compiler *requiring* scala lib on project's path, but I hope I'm wrong : )